src/main/scala/cognite/spark/v1/SttpClientBackendFactory.scala

```diff
@@ -6,7 +6,7 @@ import org.asynchttpclient.AsyncHttpClient
 import sttp.client3.SttpBackendOptions
 
 object SttpClientBackendFactory {
-  def create(prefix: String = "Cdf-Spark"): AsyncHttpClient = {
+  def create(prefix: String = "Cdf-Spark", requestTimeout: Option[Int] = None): AsyncHttpClient = {
```

Comment:
```suggestion
  def create(prefix: String = "Cdf-Spark", requestTimeoutMs: Option[Int] = None): AsyncHttpClient = {
```
or even better maybe some typed duration